[debug] [2018-02-19T10:46:10.793Z] ----------------------------------------------------------------------
[debug] [2018-02-19T10:46:10.809Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\vignesh_k05\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase login
[debug] [2018-02-19T10:46:10.809Z] CLI Version:   3.17.4
[debug] [2018-02-19T10:46:10.809Z] Platform:      win32
[debug] [2018-02-19T10:46:10.809Z] Node Version:  v7.4.0
[debug] [2018-02-19T10:46:10.809Z] Time:          Mon Feb 19 2018 16:16:10 GMT+0530 (India Standard Time)
[debug] [2018-02-19T10:46:10.809Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=312469834&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2018-02-19T10:46:32.172Z] >>> HTTP REQUEST POST https://accounts.google.com/o/oauth2/token  
 { code: '4/AADcfjSpDoxm5uIakVw5tnCWTETffxfej8L2hlbKaqZMsdy2E9_o9lYR9vI3jasLEzB-ZYUh2xcIKfWzj_GY3z4',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  redirect_uri: 'http://localhost:9005',
  grant_type: 'authorization_code' } 
 Mon Feb 19 2018 16:16:32 GMT+0530 (India Standard Time)
[debug] [2018-02-19T10:46:53.241Z] Token Fetch Error: Error
    at FirebaseError (C:\Users\vignesh_k05\AppData\Roaming\npm\node_modules\firebase-tools\lib\error.js:11:17)
    at Request._callback (C:\Users\vignesh_k05\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:34:23)
    at self.callback (C:\Users\vignesh_k05\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:186:22)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at Request.onRequestError (C:\Users\vignesh_k05\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:878:8)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:188:7)
    at TLSSocket.socketErrorListener (_http_client.js:309:9)
    at emitOne (events.js:96:13)
[debug] [2018-02-19T10:47:25.348Z] ----------------------------------------------------------------------
[debug] [2018-02-19T10:47:25.348Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\vignesh_k05\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase login
[debug] [2018-02-19T10:47:25.348Z] CLI Version:   3.17.4
[debug] [2018-02-19T10:47:25.348Z] Platform:      win32
[debug] [2018-02-19T10:47:25.348Z] Node Version:  v7.4.0
[debug] [2018-02-19T10:47:25.348Z] Time:          Mon Feb 19 2018 16:17:25 GMT+0530 (India Standard Time)
[debug] [2018-02-19T10:47:25.348Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=400399346&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2018-02-19T10:47:35.758Z] >>> HTTP REQUEST POST https://accounts.google.com/o/oauth2/token  
 { code: '4/AAAL78OzgPtz-UfD4dGzaMJI8BgW5AgD1kEjKji0oK7xG7olImI5Ves5a37uoW08-IZuXVTAcOr7vhUHkDO5XIg',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  redirect_uri: 'http://localhost:9005',
  grant_type: 'authorization_code' } 
 Mon Feb 19 2018 16:17:35 GMT+0530 (India Standard Time)
[debug] [2018-02-19T10:47:56.826Z] Token Fetch Error: Error
    at FirebaseError (C:\Users\vignesh_k05\AppData\Roaming\npm\node_modules\firebase-tools\lib\error.js:11:17)
    at Request._callback (C:\Users\vignesh_k05\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:34:23)
    at self.callback (C:\Users\vignesh_k05\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:186:22)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at Request.onRequestError (C:\Users\vignesh_k05\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:878:8)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:188:7)
    at TLSSocket.socketErrorListener (_http_client.js:309:9)
    at emitOne (events.js:96:13)
